programs/clearing_house/src/state/user.rs

```diff
@@ -130,6 +130,11 @@ pub struct MarketPosition {
     pub open_orders: u128,
     pub unsettled_pnl: i128,
 
+    // lp stuff
+    pub lp_tokens: u128,
```

Comment:
is `lp_tokens` really the best name since there are no tokens?

Reply:
not sure what else we could call it